cs/quickstart/home-page.texy

```diff
@@ -127,6 +127,10 @@ public function renderDefault(): void
 Presenter nyní obsahuje jednu renderovací metodu `renderDefault()`, která předává data z databáze do šablony (View). Šablony jsou umístěny v `app/Presenters/templates/{PresenterName}/{viewName}.latte`, takže v tomto případě je šablona umístěna v `app/Presenters/templates/Homepage/default.latte`. V šabloně nyní bude k dispozici proměnná `$posts`, ve které jsou příspěvky získané z databáze.
 
 
+.[tip]
```

Comment:
Můžeš vysvětlit, proč není ideální?

Reply:
priotože začátečníka neuuščíme používat MVC a pouze VC, i já na začátku programování v Nette před mnoha lety mělů špatné návyky, které jsem získal tu!

Reply:
Myslím vysvětlit v textu.

Reply:
máš tip, jak to sepsat?

Reply:
Tyjo teď řeším uplně jinou část dokumentuace, takže na to nejsem vůbec naladěný.